data/saas/config/mailchimp_config.yml

```diff
@@ -69,7 +69,22 @@ saas_config:
               type: query
               identity: email
               data_type: string
-          data_path: exact_matches/members
+          data_path: exact_matches.members
+        update:
+          path: /3.0/lists/<list_id>/members/<subscriber_hash>
+          request_params:
+            - name: list_id
+              type: path
```

Comment:
Are there examples of "body" request params? Can those be nested?

Reply:
I have removed "body" request_params from the SaaS config since I didn't see any use cases for this in the current legacy adaptors. We can scope this out in the future if there is a need.

Reply:
ok thanks for clarifying